dotcom-rendering/src/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -138,29 +132,28 @@ export const canShowReaderRevenueEpic = async (
 	const hasConsentForArticleCount =
 		await hasCmpConsentForWeeklyArticleCount();
 
+	const openCmp = () => {
+		cmp.showPrivacyManager();
+	};
+
 	return {
 		show: true,
 		meta: {
-			module,
-			fetchEmail,
-			hasConsentForArticleCount,
-			stage,
+			name: module.name,
+			props: {
```

Comment:
Here the client adds some extra fields to the props. Previously this was done further down in `ReaderRevenueEpic`